compact/pid/drich.xml

```diff
@@ -24,6 +24,7 @@
 <constant name="DRICH_sensorbox_length"   value="50.0*cm"/>  <!-- z-length of the extrusion -->
 <constant name="DRICH_sensorbox_rmin"     value="DRICH_rmax1 + 2*cm"/>  <!-- lower radial limit of the extrusion -->
 <constant name="DRICH_sensorbox_rmax"     value="DRICH_rmax2 + 5*cm"/>  <!-- upper radial limit of the extrusion -->
+<constant name="DRICH_sensorbox_dphi"     value="48*degree"/>  <!-- azimuthal width of the extrusion -->
```

Comment:
This is the smallest delta phi that still contains the sensors. The project has this as 42 degrees, with the 18 degrees between sensor boxes for service routing. Out of scope for this PR but the sensors placement may need adjustment.

Reply:
You can likely reduce `sphericalpatch` `phiw` to decrease the azimuthal extent that sensors are allowed; 30 deg is maximal, meaning _no_ azimuthal cut (letting its `zmin` limit the azimuthal extent)

https://github.com/eic/epic/blob/c3c0277afa1417a77050c15f3707d4789f1a259d/compact/pid/drich.xml#L331

Reply:
I'm not going to change this within this PR, but @chchatte92 we do need to change this to a smaller value that fits with what the project has in terms of azimuthal width. I don't know what it should be to fit in the 42 degrees azimuthal width, but I'll let you figure that out. For now we can acommodate this by bundling the services up in thicker bundles over a smaller delta phi.